cmd/ana/main_test.go

```
@@ -553,32 +393,53 @@ func TestStartNudge_SkipConditions(t *testing.T) {
 	}
+
+	// Disabled interval — config writes UpdateCheckInterval="0" so
+	// ParseInterval returns enabled=false and startNudge returns nil
+	// before launching the goroutine.
+	t.Run("disabled interval", func(t *testing.T) {
+		version = "1.0.0"
+		dir := t.TempDir()
+		off := "0"
+		cfg := config.Config{UpdateCheckInterval: &off}
+		path := filepath.Join(dir, "ana", "config.json")
+		if err := os.MkdirAll(filepath.Dir(path), 0o755); err != nil {
```

golangci-lint (2.11.4)

[medium] G301: Expect directory permissions to be 0750 or less

(gosec)